.github/workflows/python-scripts.yml

```diff
@@ -0,0 +1,16 @@
+name: Python scripts linters
+
+on:
+  push:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install linters
+        run: pip3 install black
+
+      - uses: actions/checkout@v2
+
+      - name: Check formatting (black)
+        run: black . --check
```

Comment:
Could we check the formatting but filter just the scripts folder?

Reply:
It would actually be `scripts` and `ci`, but yes, definitely need to do that.